src/components/FloatingHelper/FloatingHelper.protractor.driver.ts

```diff
@@ -0,0 +1,15 @@
+import {$, ElementFinder, by} from 'protractor';
+import {popoverDriverFactory} from 'wix-ui-core/dist/src/baseComponents/Popover/Popover.protractor.driver';
+import {DataHooks} from './DataHooks';
+import {helperContentDriverFactory} from './content/HelperContent.protractor.driver';
+
+export const floatingHelperDriverFactory = (element: ElementFinder) => {
+  return {
+    ...popoverDriverFactory(element),
+    getHelperContentDriver: () =>  helperContentDriverFactory(element.$(byDataHook(DataHooks.innerContent))),
+  };
+};
+
+export function byDataHook(hook: string) {
```

Comment:
Best to put it in wix-ui-test-utils

Reply:
I have an Pr in my backlog for adding this to `wix-ui-test-utils`, but its a bit complicated, and it is delayed. ONce it is there, then I'll refactor all the occurences of `data-hook`.